### PR TITLE
Return the streams from the seed file

### PR DESCRIPTION
**Pull request by nimeshgurung**
1. I have added a seed file with minimum amount of attributes needed for each stream.
2.  `/streams` api returns the data from the seed file. This should help me start integration with the frontend.

### CODE REVIEW

File: app/controllers/streams_controller.rb [hunk 1]
```diff
@@ -4,22 +4,14 @@
 # Streams API Controller
 #
 class StreamsController < ApplicationController
+  def initialize(params)
+    super(params)
+
+    file = File.read(File.join(File.dirname(__FILE__), '../../data/seedFile.json'))
```
Comment (rajgurung) [inline app/controllers/streams_controller.rb:10]: Nice one, we can make this seed_file coming from `models` layer. So `app/models/stream.rb`

Comment (rajgurung) [reply app/controllers/streams_controller.rb:10]: Also, for YML load there is a library that will handle read.

https://stackoverflow.com/questions/8878389/ruby-loading-config-yaml-file-in-same-dir-as-source/8878436

Comment (nimeshgurung) [reply app/controllers/streams_controller.rb]: I am not particularly clear on why we need to load the json file from yaml but I am going to pass on that one as it is not pretty clear to me at the moment.

Comment (rajgurung) [reply app/controllers/streams_controller.rb]: Was thinking of json as yml 🤦 , looks good as is.

**Pull request merged**